fontbe/src/cmap.rs

```diff
@@ -0,0 +1,51 @@
+//! Generates a [cmap](https://learn.microsoft.com/en-us/typography/opentype/spec/cmap) table.
+
+use fontdrasil::orchestration::Work;
+use read_fonts::types::GlyphId;
+use write_fonts::{dump_table, tables::cmap::Cmap};
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Context},
+};
+
+struct CmapWork {}
+
+pub fn create_cmap_work() -> Box<BeWork> {
+    Box::new(CmapWork {})
+}
+
+impl Work<Context, Error> for CmapWork {
+    /// Generate [cmap](https://learn.microsoft.com/en-us/typography/opentype/spec/cmap)
+    fn exec(&self, context: &Context) -> Result<(), Error> {
+        // cmap only accomodates single codepoint : glyph mappings; collect all of those
+        let static_metadata = context.ir.get_static_metadata();
```

Comment:
if you had the global {codepoint : glyph} mappings in the static metadata IR, you'd be done here

Reply:
yup! I have forkecd that discussion to https://github.com/googlefonts/fontmake-rs/issues/145.